apps/dapp/package.json

```diff
@@ -94,7 +94,8 @@
     "tsconfig-paths-webpack-plugin": "^3.5.2",
     "typechain": "^8.1.1",
     "typescript": "^4.9.4",
-    "vite": "^2.7.2"
+    "vite": "^2.7.2",
+    "vite-plugin-svgr": "^4.2.0"
```

Comment:
What command did you use to add this? I don't see any root level `yarn.lock` changes I would expect. In the monorepo we have to use a specific command to add it such as 

`yarn workspace @temple/dapp add vite-plugin-svgr`

Reply:
I think I've discard the changes on that file by mistake, thx for pointing out